### ISSUE TITLE
docker run  get Cannot find package '@/models' imported from error

### ISSUE THREAD

**Issue by yestool** [honojs/node-server#168]
I hope to use hono instead of express as a tool for developing my small website. deploy on my self vps.

My Project Structure:

```
|-- Dockerfile
|-- README.md
|-- build.ts
|-- dist
|   |-- index.js
|   |-- models
        `-- user.js
|-- package.json
|-- src
|   |-- index.ts
|   |-- models
        `-- user.ts
| -- tsconfig.json
| -- package.json
```

I did the operations described in the document：  https://hono.dev/getting-started/nodejs#dockerfile

But when I started Docker, I got an error

```
> node dist/index.js

node:internal/modules/esm/resolve:854
  throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath(base), null);
        ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@/models' imported from /root/webs/app-h/dist/index.js
    at packageResolve (node:internal/modules/esm/resolve:854:9)
    at moduleResolve (node:internal/modules/esm/resolve:927:18)
    at defaultResolve (node:internal/modules/esm/resolve:1157:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:227:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:87:39)
    at link (node:internal/modules/esm/module_job:86:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```
the index.ts is :

```
import { serve } from '@hono/node-server'
import { Hono } from 'hono'
import { createUser, sayHello } from '@/models/user';
const app = new Hono()
app.get('/', (c) => {
  return c.text('Hello Hono!'+ sayHello(createUser('John', 30)))
})
const port = 3000
console.log(`Server is running on port ${port}`)
serve({
  fetch: app.fetch,
  port
})

```

the user is:

```
export function createUser(name: string, age: number) {
  return { name, age };
}
export function sayHello(user: { name: string; age: number }) {
  return `Hello, my name is ${user.name} and I am ${user.age} years old.`;
}
```
tsconfig.json is:

```
{
  "compilerOptions": {
    "target": "ESNext",
    "module": "ESNext",
    "moduleResolution": "Bundler",
    "strict": true,
    "skipLibCheck": true,
    "outDir": "./dist",
    "types": [
      "node"
    ],
    "jsx": "react-jsx",
    "jsxImportSource": "hono/jsx",
    "baseUrl": ".",
    "paths": {
      "@/*": ["src/*"],
    }
  },
  "exclude": ["node_modules"]
}
```

and package.json ： "type": "module"

I want to know if it's the problem with my usage or is it currently not supported by the node-server ?

**Comment by yusukebe** [comment]
Hi @yestool . Hmm, this may not be a Hono matter but is related to the TypeScript project setting. And this is not a bug report, so it is proper to add the comment on the [Disccusion](https://github.com/orgs/honojs/discussions).

**Comment by yestool** [comment]
Hi @yusukebe  I solved it. I'm a typescript beginner and I don't know if this is the right thing to do, but it works fine now.
This is an example code repository ：   [hono-nodejs](https://github.com/yestool/hono-nodejs)  

I introduced `tsc-alias`  support.

I'm looking forward to more official docker instances example. for deploying hono programs on self vps.

Thank you very much , hono is very useful and very easy. 🌹 😘
